playwright-e2e/tests/page-formating-tests.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { Page, expect, test } from '@playwright/test';
+
+import { urlConfig } from '../utils/urls';
+
+test.describe('Start application details e2e journey in English', () => {
+  let page: Page;
+  test.beforeEach(async ({ browser }) => {
+    const context = await browser.newContext({
+      locale: 'en',
+    });
+    page = await context.newPage();
+    await page.goto(urlConfig.citizenStartUrl);
+  });
+  test('Start up jourey url is equal to the initial url', async () => {
+    await expect(page).toHaveURL(urlConfig.citizenStartUrl);
+  });
+
+  test('Start up journey page is default to English', async () => {
+    const htmlLang = await page.evaluate(() => document.documentElement.lang);
+
+    expect(htmlLang).toBe('en');
+  });
+  test('Start up journey page is to have Cymraeg language link to be visible', async () => {
+    const linkLocator = page.locator('text=cymraeg');
+    await expect(linkLocator).toBeVisible();
+  });
+  test('Start up journey page clicking Cymraeg langauge changes html language to cy', async () => {
+    const linkLocator = page.locator('text=cymraeg');
+    await linkLocator.click();
+    await page.waitForLoadState('load');
+
+    const htmlLang = await page.evaluate(() => document.documentElement.lang);
+
+    expect(htmlLang).toBe('cy');
```

Comment:
might be worth checking if you continue on the user journey, the language remains welsh and does not change back to english